RELEASE_NOTES.md

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+v1.0.0-alpha.X
+--------------
+
+### Breaking changes
+
+- Disabled the (nascent) C bindings by default. To enable, the
+  `OPENASSETIO_ENABLE_C` CMake option must be explicitly set to `TRUE`.
```

Comment:
[Consider][Minor] Rather than `TRUE`, should we say `ON`, as that's the vernacular used in the scripts?

Reply:
Yeah I think you're right. My thinking at the time was that `TRUE` is a bit less CMake-specific to may be a bit more readable, but I think consistency with the rest of the code is more valuable.

Reply:
Updated and force-pushed, rather than fixupped, to b56c7209